babel.config.js

```diff
@@ -54,8 +68,6 @@ module.exports = {
   plugins: [
     'babel-plugin-optimize-clsx',
     ['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
This is something we need to keep an eye on. It's already pretty dangerous to use stage-2 (at the time) proposals. Using those in loose mode however should never be an option for library code. It makes sense for app code but if you don't control the input we should default to strict by default.

In the end (I would think) we want to get rid of these transforms in favor of less bundle size for modern browsers. Using non-spec compliant language features should require a strong argument for it.

Reply:
An additional thought, with the move to functional components we might not need this transform after the migration has finished.

Reply:
Yeah but excess transforms never hurt. Even if we "accidentally" add class properties into library code we only end up shipping a little bit more bytes than necessary which is better than crashing IMO. It's also required for docs code. I don't think we need to convert those too.